viewing_party/party.py

```diff
@@ -0,0 +1,117 @@
+def create_movie(movie_title, genre, rating):
+    new_movie = { }
+    if movie_title==None or genre==None or rating==None:
+        return None
+    new_movie['title'] = movie_title
+    new_movie['genre'] = genre
+    new_movie['rating'] = rating
+    return new_movie
+
+def add_to_watched(user_data, movie):
```

Comment:
great implementation! you could also make it a one-liner 

`user_data["watched"].append(movie)

    return user_data`